controls/pcidss_4_ocp4.yml

```diff
@@ -3258,20 +3262,20 @@ controls:
         by providing access or evidence that comparable technical testing has been undertaken.
       levels:
         - base
-      status: pending
+      status: not applicable
 
   - id: '11.5'
     title: Network intrusions and unexpected file changes are detected and responded to.
     levels:
       - base
-    status: pending
+    status: automated
     controls:
     - id: 11.5.1
       title: Intrusion-detection and/or intrusion-prevention techniques are used to detect and/or
         prevent intrusions into the network.
       levels:
         - base
-      status: pending
```

Comment:
I wonder if [acs_sensor_exists](https://github.com/ComplianceAsCode/content/tree/master/applications/openshift/general/acs_sensor_exists) fits here since it support monitoring on network side of things

Reply:
Looks like it's included as part of 11.5.1.1 on line 3298?